compiler/rustc_codegen_cranelift/src/base.rs

```diff
@@ -38,9 +38,8 @@ pub(crate) fn codegen_fn<'tcx>(
 
     // Predefine blocks
     let start_block = bcx.create_block();
-    let block_map: IndexVec<BasicBlock, Block> = (0..mir.basic_blocks().len())
-        .map(|_| bcx.create_block())
-        .collect();
+    let block_map: IndexVec<BasicBlock, Block> =
+        (0..mir.basic_blocks().len()).map(|_| bcx.create_block()).collect();
```

Comment:
Please don't just reformat cg_clif. This will conflict badly when doing a sync. These format changes are probably caused by the different rustfmt config used by cg_clif and rustc.

Reply:
oh wow timing, I just contacted you on zulip about it seconds before you wrote this 😆 

I did already revert those formatting changes locally, this happend accidentally as I didn't realize I had rustfmt turned on everywhere